config/domains.go

```diff
@@ -63,6 +74,15 @@ func (d Domain) GetCustomParams() url.Values {
 	return url.Values{}
 }
 
+// ASCII converts the name of [Domain] to its ASCII form,
+// using non-transitional process specified in UTS 46.
+//
+// Note: conversion errors are silently discarded and partial conversion
+// results are used.
+func (d *Domain) ASCII() {
```

Comment:
Rename function  name as `ToASCII`?

Reply:
Rename if we want to create a copy instead of editing in-place, otherwise do nothing.